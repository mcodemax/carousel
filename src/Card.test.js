import React from "react";
import { render, fireEvent } from "@testing-library/react"; //but none is installed. You must install peer dependencies yourself. npm warning after installing react testing library
import Card from "./Card";

it("should render", () => {
  render(<Card/>)
});

it("matches snapshot", () => {
    const obj = {
                    caption:"Tst Caption",
                    src: "www.google.com",
                    currNum: 5,
                    totalNum: 10
                };
    const { asFragment } = render(<Card {...obj}/>);
    expect(asFragment()).toMatchSnapshot();
});

