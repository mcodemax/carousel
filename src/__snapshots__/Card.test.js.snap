// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Tst Caption
    </h4>
    <img
      alt="Tst Caption"
      class="Card-image"
      src="www.google.com"
    />
    <small
      class="Card-small"
    >
      Image 5 of 10.
    </small>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot 2`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Tst Caption
    </h4>
    <img
      alt="Tst Caption"
      class="Card-image"
      src="www.google.com"
    />
    <small
      class="Card-small"
    >
      Image 5 of 10.
    </small>
  </div>
</DocumentFragment>
`;
